Libraries/Components/AccessibilityInfo/AccessibilityInfo.ios.js

```diff
@@ -97,15 +99,30 @@ var AccessibilityInfo = {
    * - `change`: Fires when the state of the screen reader changes. The argument
    *   to the event handler is a boolean. The boolean is `true` when a screen
    *   reader is enabled and `false` otherwise.
+   * - `announcementFinished`: iOS-only event. Fires when the screen reader has
+   *   finished making an announcement. The argument to the event handler is a dictionary
+   *   with these keys:
+   *     - `announcement`: The string announced by the screen reader.
+   *     - `success`: A boolean indicating whether the announcement was successfully made.
    */
   addEventListener: function (
     eventName: ChangeEventName,
     handler: Function
   ): Object {
-    var listener = RCTDeviceEventEmitter.addListener(
-      VOICE_OVER_EVENT,
-      handler
-    );
+    var listener;
+
+    if (eventName === 'change') {
```

Comment:
We have to unify native event names with JS ones, this will drastically simplify this code fragment.